deps/wazuh_testing/wazuh_testing/qa_ctl/provisioning/qa_provisioning.py

```diff
@@ -105,6 +105,20 @@ def __process_config_data(self, host_provision_info):
             wazuh_branch = 'master' if 'wazuh_branch' not in deploy_info else deploy_info['wazuh_branch']
             s3_package_url = None if 's3_package_url' not in deploy_info \
                                         else deploy_info['s3_package_url']
+
+            version = None if 'version' not in deploy_info \
+                                else deploy_info['version']
+            repository = None if 'repository' not in deploy_info \
+                                else deploy_info['repository']
+
+            revision = None if 'revision' not in deploy_info \
+                                else deploy_info['revision']
+
+            system = None if 'system' not in deploy_info \
+                                else deploy_info['system']
+            architecture = None if 'architecture' not in deploy_info \
```

Comment:
```suggestion
            architecture = `x86_64` if 'architecture' not in deploy_info \
```

Reply:
This requested change can be reviewed in the commit : [5c3a34d2757275d66bb9951667083965a0452fab](https://github.com/wazuh/wazuh-qa/commit/5c3a34d2757275d66bb9951667083965a0452fab)